src/EventListener/DcaListener.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of terminal42/contao-inserttags.
+ *
+ * (c) terminal42
+ *
+ * @license MIT
+ */
+
+namespace Terminal42\InsertTagsBundle\EventListener;
+
+use Contao\Backend;
+use Contao\CoreBundle\ServiceAnnotation\Callback;
+use Contao\DataContainer;
+use Contao\Image;
+use Contao\Input;
+use Contao\StringUtil;
+use Doctrine\DBAL\Connection;
+
+class DcaListener
+{
+    public function __construct(private Connection $connection) {}
+
+    /**
+     * @Callback(table="tl_inserttags", target="config.onload")
+     */
+    public function onLoadCallback(DataContainer $dc): void
```

Comment:
Now that the order is not relevant anymore, we don't need to have a tree view with all those callbacks anymore, do we?